src/nodes.coffee

```diff
@@ -1654,7 +1654,7 @@ exports.JSXExpressionContainer = class JSXExpressionContainer extends Base
 
   astProperties: (o) ->
     return
-      expression: @expression.ast o
+      expression: astAsBlockIfNeeded @expression, o
```

Comment:
The other issue was that "blocks" for JSX attribute expression values/string interpolation values were generating `SequenceExpression` AST's rather than `BlockStatement`'s (`SequenceExpression`'s should only be generated when there's actually a "sequence" in the source eg `(a; b)`)

So for example this:
```
<div a={
  b
  c
} />
```
or this:
```
a "#{
  b
  c
}"
```

So use a shared `astAsBlockIfNeeded()` helper that ensures that a `BlockStatement` AST gets generated if it's looking at a multi-statement block